app/views/layouts/application.html.erb

```diff
@@ -12,10 +12,11 @@
   </head>
 
   <body>
-
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
+    <div class="min-hight">
```

Comment:
typo here

Reply:
Actually, not a typo as you use the same spelling in the css but just to be sure we don't have issues down the line you might want to update both as that's not how you spell "height" :)